src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const inputs = document.querySelectorAll('input');
+const inputClass = 'field-label';
+const firstLatterUp = (word = '') => {
+  if (word.length > 0) {
+    return (
+      word.charAt(0).toUpperCase() + word.slice(1)
+    );
+  } else {
+    return word;
+  }
```

Comment:
```suggestion
  if (word.length > 0) {
    return (
      word.charAt(0).toUpperCase() + word.slice(1)
    );
  }
  
  return word;
```
No need for `else`. Your code will not go further after `return` anyway